axelor-account/src/main/resources/i18n/messages_en.csv

```diff
@@ -738,7 +738,7 @@
 "Amount excluding taxes",,,
 "Amount exported for direct debit",,,
 "Amount for each line",,,
-"Amount in Acc. currency",,,
+"Amount in Acc. currency","Amount in move currency",,

```

Comment:
To avoid future confusion, please change the key value instead of adding a translation